app/src/main/res/layout/list_item_location.xml

```diff
@@ -27,7 +27,7 @@
             android:layout_height="wrap_content"
             android:layout_marginLeft="12dp"
             android:layout_weight="1"
-            android:text="North offset 24 degree"
+            android:text="@string/north_offset_24_degree"
```

Comment:
This string is just a mockup, it doesn't need to be translated. Please either leave it as it was or delete this attribute altogether.